settings/src/components/email-address.js

```diff
@@ -0,0 +1,94 @@
+
+/**
+ * WordPress dependencies
+ */
+import { Button, TextControl, Notice, Spinner } from '@wordpress/components';
+import { useState, useEffect } from '@wordpress/element';
+import { useSelect } from '@wordpress/data';
+import { store as coreDataStore, useEntityRecord } from '@wordpress/core-data';
+
+/**
+ * Render the Email setting.
+ */
+export default function EmailAddress( { userId } ) {
+	const { record, edit, save, editedRecord, hasEdits } = useEntityRecord( 'root', 'user', userId );
+
+	// useEntityRecord() doesn't expose isSaving. Huh.
+	const isSaving = useSelect( ( select ) => select( coreDataStore ).isSavingEntityRecord( 'root', 'user', userId ) );
```

Comment:
@adamziel is there a way to get this from `useEntityRecord`? I don't see one either.

Reply:
For now, I've polyfill'd it in with a wrapping function in c1465bd2580fc7656af78ed3c49078be2ce03c3f which seems to work for me.